packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -664,6 +664,7 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
           aria-label={ariaLabel}
           aria-labelledby={label && !ariaLabel ? this._labelId : undefined}
           aria-multiselectable={multiSelect}
+          shouldFocusOnMount
```

Comment:
it still seems to be focusing on the close button on open for me. I don't think that's the worst thing in the world, but I think the panel focus is happening after the focuszone focus and winning.

Reply:
I've changed my approach, let me know if it still doesn't work.